From 3bd6db0b8f60143836f5a933f35b15d764fbc7dc Mon Sep 17 00:00:00 2001
From: Ludovic Rousseau <ludovic.rousseau@gmail.com>
Date: Wed, 9 May 2012 09:13:41 +0200
Subject: [PATCH 14/28] examples/Makefile.am: Use the implicit _SOURCES rule

By default automake looks for a source file foo.c for a foo binary.

libusbx.git commit 7e52e1e7534ead3f66b13bdcb28ac8587ff47ab0
---
 examples/Makefile.am |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/examples/Makefile.am b/examples/Makefile.am
index ac9132d..4545ed4 100644
--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -2,14 +2,10 @@ INCLUDES = -I$(top_srcdir)/libusb
 LDADD = ../libusb/libusb-1.0.la
 noinst_PROGRAMS = listdevs
 
-listdevs_SOURCES = listdevs.c
-
 if HAVE_SIGACTION
-dpfp_SOURCES = dpfp.c
 noinst_PROGRAMS += dpfp
 
 if THREADS_POSIX
-dpfp_threaded_SOURCES = dpfp_threaded.c
 dpfp_threaded_CFLAGS = $(THREAD_CFLAGS) $(AM_CFLAGS)
 noinst_PROGRAMS += dpfp_threaded
 endif
-- 
1.7.9.5

